.drone.yml.sig

```diff
@@ -1 +0,0 @@
-eyJhbGciOiJIUzI1NiJ9.cGlwZWxpbmU6CiAgdmFsaWRhdGU6CiAgICBpbWFnZTogcXVheS5pby91a2hvbWVvZmZpY2VkaWdpdGFsL2FjcC10b29sc2V0OmxhdGVzdAogICAgcHVsbDogdHJ1ZQogICAgY29tbWFuZHM6CiAgICAtIC9wbGF0Zm9ybS9zY3JpcHRzL3RmLXZhbGlkYXRlLnNoCiAgICB3aGVuOgogICAgICBldmVudDogcHVsbF9yZXF1ZXN0Cg.IPTqit6vB0jMWY4ClB9_7d-Nfr1ljEdCDCPJJvRNrec
```

Comment:
It looks like the file contents are empty rather than the file being deleted completely, can you confirm?

Reply:
Ignore :D